src/main.controller.js

```diff
@@ -830,7 +830,7 @@ plugin.controller('wgnConfigCtrl', ['$scope', '$q', '$routeParams', 'znData', 'z
 		 * @returns {Object} the updated (and now valid) filter
 		 */
 		function reconstructFilter (filter, config) {
-			let newFilter = angular.extend({}, filter);
```

Comment:
@tehpsalmist this is actually more suited as a `chore` or `style` commit!

here's why:
- it doesnt fix a bug
- even if you argued that it might fight a potential bug on IE9 or whatever, it was a bug introduced 3 commits ago  which has never seen the light of day... so it makes no sense to have this added in the changelog

in this case its fine i dont care but just a heads up :)

Reply:
Thanks for the explanation!